jsk_robot_common/jsk_robot_utils/euslisp/robot-interface-utils.l

```diff
@@ -0,0 +1,32 @@
+(defun robot-file (name)
+  "You can get robot-interface file according to `name' argument.
+You can create `*ri*' like 
+(progn (load (robot-file (ros::get-param \"/robot\")))
```

Comment:
please follow https://github.com/ros-infrastructure/rep/pull/104/files and use standard param name for robot type, currently `robot/type` is suggested.